phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -692,6 +692,17 @@ private boolean extractTablesAndIndexes(PhoenixConnection conn) {
         return viewIndexes;
     }
 
+    @VisibleForTesting
+    public String getViewIndexesSql(String viewName, String schemaName, String tenantId) {
+        return "SELECT DISTINCT COLUMN_FAMILY FROM "
+                + "SYSTEM.CATALOG "
+                + "WHERE TABLE_NAME = \'" + viewName + "\'"
+                + (!StringUtil.EMPTY_STRING.equals(schemaName) ? "AND TABLE_SCHEM = \'"
+                + schemaName + "\' " : "")
+                + "AND LINK_TYPE = " + PTable.LinkType.INDEX_TABLE.getSerializedValue()
```

Comment:
Likewise, add space before AND

Reply:
There is a space after quote but I think before AND makes it more readable. ll change it.